docs/code-examples/expressions/jmespath.md

```diff
@@ -225,3 +174,53 @@ Use multiselect list to get the first and last names and create new lists contai
 ```
 
 
+
+### An alternative to arrow functions
+
+You can't use arrow functions in expressions. This means you can't easily do things like looking for an item with a specific key.
+
+For example, given the below input data, you could do a search like "find the item with the name Lenovo and tell me their category ID."
+
+```js
+return[
```

Comment:
I would return both items underneath the `json` property. It for sure works like that, as n8n dos now add it automatically, but it would then make the example that follows clearer (as it also uses `json`).